src/SpecialEditionBanner/index.tsx

```diff
@@ -2,22 +2,26 @@ import React from 'react';
 import { AppBanner } from '../AppBanner';
 import type { OphanComponentEvent } from '@guardian/types';
 import type { BrazeClickHandler } from '../utils/tracking';
+import { BrazeComponent } from '../BrazeMessageComponent';
+import { COMPONENT_NAME } from './canRender';
+export { COMPONENT_NAME };
```

Comment:
If we need to export this, shall we combine with the export at the bottom of the file?

Reply:
Given we're no longer exporting the component at the bottom of the file, keeping this export next to the import I think is easier to follow.